static/js/settings_profile_fields.js

```diff
@@ -344,6 +373,62 @@ function open_edit_form(e) {
     });
 }
 
+// Display custom profile fields in user profile popover limit check,
+// if its crosses or equals the limit we are disabling option for
+// display custom profile field in user profile popover.
+function update_profile_fields_checkboxes() {
+    if (page_params.display_in_profile_popover_fields_count >= 2) {
+        $("#add-new-profile-field #profile_field_display_in_profile_popover").prop(
+            "disabled",
+            true,
+        );
+        $("#add-new-profile-field .checkbox").addClass("tippy-zulip-tooltip");
+        // Disabling only uncheck checkboxes in table, so user should able uncheck checked checkboxes.
+        $("#admin_profile_fields_table .display_in_profile_popover_checkbox_false").prop(
+            "disabled",
+            true,
+        );
+        $("#admin_profile_fields_table .display_in_profile_popover_false").addClass(
+            "tippy-zulip-tooltip",
+        );
+    } else {
+        $("#add-new-profile-field #profile_field_display_in_profile_popover").prop(
+            "disabled",
+            false,
+        );
+        $("#add-new-profile-field .checkbox").removeClass("tippy-zulip-tooltip");
+        $("#admin_profile_fields_table .display_in_profile_popover_checkbox_false").prop(
+            "disabled",
+            false,
+        );
+        $("#admin_profile_fields_table .display_in_profile_popover_false").removeClass(
+            "tippy-zulip-tooltip",
+        );
+    }
```

Comment:
These duplicate branches are unnecessary. Create a boolean `multiple = page_params.display_in_profile_popover_fields_count >= 2` and use `.prop("disabled", multiple)` and `.toggleClass("tippy-zulip-tooltip", multiple)`.

Reply:
Okay fine, I was trying to write easy-to-read code.

Reply:
After using this method, I am getting a test error while running test-js-with-node.
```
test failed: /srv/zulip/frontend_tests/node_tests/settings_profile_fields.js > populate_profile_fields

Error: You must create a stub for $("#add-new-profile-field .checkbox").toggleClass
    at Object.get (/srv/zulip/frontend_tests/zjsunit/zjquery.js:72:27)
    at update_profile_fields_checkboxes (/srv/zulip/static/js/settings_profile_fields.js:386:5)
    at Object.do_populate_profile_fields (/srv/zulip/static/js/settings_profile_fields.js:502:5)
    at test_populate (/srv/zulip/frontend_tests/node_tests/settings_profile_fields.js:64:29)
    at /srv/zulip/frontend_tests/node_tests/settings_profile_fields.js:199:5
    at /srv/zulip/frontend_tests/zjsunit/test.js:34:13
    at Object.exports.with_overrides (/srv/zulip/frontend_tests/zjsunit/namespace.js:490:9)
    at exports.run_test (/srv/zulip/frontend_tests/zjsunit/test.js:33:19)
    at Object.<anonymous> (/srv/zulip/frontend_tests/node_tests/settings_profile_fields.js:71:1)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Module._compile (/srv/zulip-npm-cache/18b43a2b2537b651d00677b0f92d76223b4cfd3a/node_modules/pirates/lib/index.js:136:24)
    at Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Object.newLoader [as .js] (/srv/zulip-npm-cache/18b43a2b2537b651d00677b0f92d76223b4cfd3a/node_modules/pirates/lib/index.js:141:7)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.load [as _load] (/srv/zulip/frontend_tests/zjsunit/namespace.js:90:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at run_one_module (/srv/zulip/frontend_tests/zjsunit/index.js:78:5)
(...)

```
I am just confused about where and what test code should write in [node_tests/settings_profile_fields.js](https://github.com/zulip/zulip/blob/32d2c53c62fc6349a42f630508a8919fe148e731/frontend_tests/node_tests/settings_profile_fields.js), because I didn't saw any code related to testing jquery code in this file, thank you.